.gitignore

```diff
@@ -1,3 +1,19 @@
+# Whitelist dotfiles
+.*
```

Comment:
Wouldn't it be better to use `/.*` ? I think that way we wouldn't have to whitelist subdirectories below.